scripts/EdgePy.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python3
```

Comment:
If you name the file `edgepy` then it can be used on the CLI like this:

```bash
❯ edgepy --options --flags > results.txt
```

Instead of:
```bash
❯ EdgePy.py --options --flags > results.txt
```

No one likes to:

1. Type capitals.
2. Type language extensions (the language should be abstracted)

Reply:
Doesn't bother me - I didn't create the script.  (-:

Reply:
Oh,  Completely misunderstood what you were saying.  removed the shebang.  We can move this to another location - we need a discussion on structure.